src/test/resources/application-test.yml

```diff
@@ -6,6 +6,8 @@ spring:
   codec:
     max-in-memory-size: 10MB
 
+spring.cache.type: none
```

Comment:
Was getting lots of errors since JCache doesn't play nicely with integration tests and it was complaining that the cache items were already registered.  Googling it suggested that either I set the `type` to `none` here or I add `DirtiesContext` to all the tests.  Gone for the former, but will need to add some tests for the caching as all now bypass it.